src/instrumentations/external/kafkajs/propagator.ts

```diff
@@ -22,7 +22,19 @@ adding toString() to make sure string is returned
 */
 export const bufferTextMapGetter: TextMapGetter = {
   get(carrier, key) {
-    return carrier?.[key]?.toString();
+    if (carrier === undefined) {
```

Comment:
```suggestion
    if (!carrier) {
```

Tbh any non-object value is probably unexpected

Reply:
:handshake: